SamuelMtzA.md

SamuelMtzA opened the pull request:
---
quote: Nunca pares de aprender 
---

lasr21 reviewed (changes requested):
**Hola SamuelMtzA,**
**Estamos muy felcies de verte en Talent Land, vamos a revisar ese PR!**


### Tengo algunos elementos de los que necesito que te ocupes antes de que podamos aceptar este PR:

* *Uh Oh! He encontrado algunos problemas con el lugar donde creaste tus archivos!* 
	*El archivo .md requerido no existe, asegúrete de que el archivo `data/SamuelMtzA.md` existe*
*Asegúrete de que todos los cambios estén incluidos en el directorio`data/`. Invalid file paths:* 

	* `.github/workflows/src/index.js`
	* `README.md`


Siéntete libre de volver a solicitarme una revisión y volveré y echaré un vistazo.

Commit:
Merge pull request #127 from Codex-216/patch-1

Create Codex-216.md

Commit:
Merge pull request #126 from jona866/patch-1

Create jona866.md

germanandres3 committed:
Oña

Commit:
Merge pull request #123 from ChrisE-Vazq/patch-6

Create  ChrisE-Vazq.md

Yiro13 committed:
Create Yiro13.md

Commit:
Merge pull request #122 from SamuelIMR/patch-1

Create SamuelIMR.md

Commit:
Merge pull request #121 from JuanRaV/patch-1

Create JuanRaV.md

Commit:
Merge pull request #120 from Chemin7/patch-2

Create Chemin7.md

Commit:
Merge pull request #117 from christian-munoz-rdz/patch-2

Create christian-munoz-rdz.md

Commit:
Merge pull request #114 from rositaa-as/patch-1

Create rositaa-as.md

Commit:
Merge pull request #112 from AngelLego/patch-1

Create AngelLego.md

Commit:
Merge pull request #228 from Chrisburg22/patch-2

Create Chrisburg22.md

Commit:
Merge pull request #229 from Yiro13/patch-1

Create Yiro13.md

Commit:
Merge pull request #108 from charlesv39/patch-2

Create charlesv39.md

IsoleucineDev committed:
Create Ileana-B4.md

KillerSkull21 committed:
Create KillerSkull21.md

OROC-999 committed:
Create OROC-999.md

Commit:
Merge pull request #226 from germanandres3/patch-1

Create germanandres3.md

Commit:
Merge pull request #232 from KillerSkull21/patch-3

Create KillerSkull21.md

Commit:
Merge pull request #231 from Ileana-B4/patch-6

Create Ileana-B4.md

Commit:
Merge pull request #80 from jnvr1/patch-3

Create jnvr1.md

Commit:
Merge pull request #84 from ItielSanzAXO/patch-1

Create ItielSanzAXO.md

OROC-999 committed:
Hola

Commit:
Merge pull request #86 from christian0919/patch-2

Create christian0919.md

zickfreak committed:
Create Zickfreak.md

Commit:
Merge pull request #233 from OROC-999/patch-1

Create OROC-999.md

zickfreak committed:
Rename Zickfreak.md to zickfreak.md

Commit:
Merge pull request #104 from JaviCandia/patch-2

Create JaviCandia.md

Commit:
Merge pull request #97 from JAFB321/patch-2

Create JAFB321.md

Commit:
Merge pull request #235 from zickfreak/patch-1

Create Zickfreak.md

Commit:
Merge pull request #79 from Eliqn/patch-1

Create Eliqn.md

Commit:
Merge pull request #78 from JoseFarfandl/patch-6

Create JoseFarfandl.md

MexaGit committed:
Talent Land 2023 - last pull

Participa para ganar swag especial que tenemos para ti en por mandar un pull request.

AlexisAndresHR committed:
Pull request AlexisAndresHR

ShannenHeredia committed:
Aqui vamos de nuevo

Olvide contar los caracteres XD, upsis

henryalejo committed:
henryalejo PR

AndreAlmanza committed:
AndreAlmanza.md

henryalejo committed:
Merge pull request #1 from henryalejo/henryalejo

henryalejo PR

Commit:
Merge pull request #280 from henryalejo/main

henryalejo PR

William-Lugo committed:
Create William-lugo.md

William-Lugo committed:
Update William-lugo.md

Commit:
Merge pull request #289 from William-Lugo/patch-1

DEmmaRL committed:
Create DEmmaRL.md

Commit:
Merge pull request #299 from DEmmaRL/patch-2

Itachi1308 committed:
Update and rename Itachi1308 to Itachi1308.md

Alexfeelme committed:
Create <Alexfeelme>.md

Commit:
Merge pull request #305 from Itachi1308/patch-4

Alexfeelme committed:
Rename <Alexfeelme>.md to Alexfeelme.md

Commit:
Merge pull request #308 from Alexfeelme/patch-1

Create <Alexfeelme>.md

Francisco-Frias committed:
TEKERone.md

Commit:
Merge pull request #253 from ShannenHeredia/patch-2

CarlosNunezMX committed:
CarlosNunezMX 👋

lalodsi committed:
Create lalodsi.md

Commit:
Merge pull request #320 from lalodsi/patch-1

Commit:
Merge pull request #319 from CarlosNunezMX/patch-1

theaaronn committed:
Create 1Aaron1

Commit:
Merge pull request #321 from 1Aaron1/patch-1

AlanJuarezF committed:
Create AlanJuarezF.md

Commit:
Merge pull request #322 from AlanJuarezF/patch-1

AlexhHr23 committed:
Create AlexhHr.md

Commit:
Merge pull request #323 from AlexhHr23/patch-1

Commit:
Merge pull request #318 from TEKERone/patch-2

Gus-Barrera committed:
Create Gus-Barrera

Commit:
Merge pull request #325 from Gus-Barrera/patch-1

CarlosJair117 committed:
Create CarlosJair117

Commit:
Merge pull request #327 from CarlosJair117/patch-1

Andrewar08 committed:
Create Andrewar08.md

Commit:
Merge pull request #331 from Andrewar08/patch-3

MikeRbl committed:
Create MikeRbl.md

Commit:
Merge pull request #243 from AlexisAndresHR/reto-aahr

Commit:
Merge pull request #332 from MikeRbl/patch-2

Create MikeRbl.md

BlastyFN committed:
Create BlastyFN.md

lorenzanav committed:
lorenza's commit

Commit:
Merge pull request #334 from BlastyFN/main

Commit:
Merge pull request #335 from lorenzanav/main

AligheriD committed:
Create AligheriD.md

Commit:
Merge pull request #336 from AligheriD/patch-1

sly8057 committed:
Create WINniePooh1912.md

EymiVasquez committed:
Create EymiVasquez.md

Commit:
Merge pull request #339 from EymiVasquez/patch-1

adpadillar committed:
my pr

NicoleArana committed:
Create NicoleArana.md

Commit:
Merge pull request #341 from adpadillar/main

Commit:
Merge pull request #342 from NicoleArana/patch-1

hfloresb111 committed:
Create hforesb111.md

Commit:
Merge pull request #344 from hfloresb111/patch-1

OscarRangelM committed:
Create OscarRangelM.md

FraVel97 committed:
Create FraVel97. Md

Commit:
Merge pull request #346 from FraVel97/patch-1

Commit:
Merge pull request #345 from OscarRangelM/patch-1

Commit:
Create robertocalderon.md

Commit:
Merge pull request #352 from robertocalderon/main

Create robertocalderon.md

fercats99 committed:
Create fercats99.md

Commit:
Merge pull request #338 from WINniePooh1912/patch-2

Create WINniePooh1912.md

jonathankof committed:
Create jonathankof.md

Kozumz committed:
Kozumz.md

Commit:
Merge pull request #357 from fercats99/patch-1

Create fercats99.md

AsuJuve committed:
Create AsuJuve.md

mattsai committed:
Create mattssai.md

imR2D2 committed:
Create imR2D2.md

Commit:
Merge pull request #359 from jonathankof/patch-2

Create jonathankof.md

NotStarGazerO committed:
Create NotStarGazerO.md

Commit:
Merge pull request #361 from AsuJuve/main

Create AsuJuve.md

Commit:
Merge pull request #363 from imR2D2/patch-1

Create imR2D2.md

Commit:
Merge pull request #365 from NotStarGazerO/patch-1

Create NotStarGazerO.md

mattsai committed:
Rename mattssai.md to mattsai.md

Commit:
Create YasserAlvarezEsqueda.md

Commit:
Merge pull request #366 from mattsai/mattsai-patch-1

Reto

CristopherSa9 committed:
Create CristopherSa9.md

Blackguz committed:
Create Blackguz.md

Commit:
Update YasserAlvarezEsqueda.md

Commit:
Merge pull request #369 from CristopherSa9/patch-1

Create CristopherSa9.md

Commit:
Merge pull request #368 from YasserAlvarezEsqueda/patch-3

Create YasserAlvarezEsqueda.md

Commit:
Merge pull request #370 from Blackguz/main

JezhYacotu committed:
Create JezhYacotu.md

rocioale1 committed:
Create rocioale1.md

abigail-marquez committed:
Create abigail-marquez.md

adanulisesorlando committed:
Create adanulisesorlando.md

Commit:
Merge pull request #372 from JezhYacotu/patch-2

Create JezhYacotu.md

Commit:
Merge pull request #373 from rocioale1/patch-2

Commit:
Merge pull request #374 from abigail-marquez/patch-1

Create abigail-marquez.md

ReivajGP committed:
Create ReivajGP.md

Commit:
Merge pull request #375 from adanulisesorlando/patch-1

Create adanulisesorlando.md

ReivajGP committed:
Update ReivajGP.md

Commit:
Merge pull request #376 from ReivajGP/patch-1

ChBrus committed:
Create ChBrus.md

Reto GitHub

oskrjfc95 committed:
Create oskrjfc95.md

Cesario1234 committed:
Create Cesario1234.md

ChBrus committed:
I made a mistake

Commit:
Merge pull request #379 from oskrjfc95/patch-1

Create oskrjfc95.md

Commit:
Merge pull request #381 from Cesario1234/patch-1

Create Cesario1234.md

Commit:
Merge pull request #241 from drSilence88/patch-3

Talent Land 2023 - last pull

Commit:
Merge pull request #278 from AndreAlmanza/patch-2

AndreAlmanza.md

Commit:
Merge pull request #378 from ChBrus/patch-1

Create ChBrus.md

Amaya120 committed:
Create Amaya120.md

jhonatanolagued committed:
Create jhonatanolagued.md

frankcarrillo committed:
Create frankcarrillo.md

Commit:
Merge pull request #360 from Kozumz/patch-4

Kozumz.md

Commit:
Merge pull request #384 from frankcarrillo/patch-1

Create frankcarrillo.md

Nefonfo committed:
Create Nefonfo.md

Commit:
Merge pull request #385 from Amaya120/patch-2

VtapiaT02 committed:
Create VtapiaT02.md

MarioLeguizamo committed:
Create MarioLeguizamo.md

Commit:
Merge pull request #390 from Nefonfo/patch-1

Create Nefonfo.md

Commit:
Merge pull request #391 from VtapiaT02/patch-1

Create VtapiaT02.md

Commit:
Merge pull request #382 from jhonatanolagued/patch-4

Create jhonatanolagued.md

Chinoiis committed:
Create Chinoiis.md

Commit:
Merge pull request #393 from MarioLeguizamo/patch-1

Create MarioLeguizamo.md

karyroo committed:
Create karyroo.md

Talend land

Commit:
Merge pull request #394 from Chinoiis/patch-2

Commit:
Merge pull request #396 from karyroo/patch-2

Victor4286 committed:
Create Víctor4286.md

ChadluPE committed:
Create ChadluPE.md

UniqueIsem committed:
Create UniqueIsem.md

Victor4286 committed:
Rename Víctor4286.md to Victor4286.md

Commit:
Merge pull request #401 from ChadluPE/patch-1

Commit:
Merge pull request #402 from UniqueIsem/patch-3

Commit:
Merge pull request #399 from Victor4286/patch-8

lissethamc committed:
Create lissethamc.md

spiegelin committed:
Create Spiegelin.md

Commit:
Merge pull request #414 from Spiegelin/patch-1

Create Spiegelin.md

albergut committed:
Create albergut.md

BrUn3y committed:
Create brun3y.md

BrUn3y committed:
Rename brun3y.md to BrUn3y.md

DannFN committed:
Add: Agregado mi archivo md

Commit:
Update albergut.md

Commit:
Merge pull request #415 from albergut/patch-1

Commit:
Merge pull request #418 from DannFN/DannFLoco

Commit:
Merge pull request #416 from BrUn3y/patch-4

PixeliGer committed:
Create PixeliGer.md

Commit:
Merge pull request #419 from PixeliGer/patch-1

AngeLuna021 committed:
Create AngeLuna021

Commit:
Merge pull request #421 from AngeLuna021/patch-2

Create AngeLuna021

edson150bpm committed:
Create edson150bpm.md

Commit:
Merge pull request #423 from edson150bpm/patch-2

Create edson150bpm.md

GorudenOsiris committed:
Create GorudenOsiris.md

devcifuentes committed:
Creat devcifuentes.md

Commit:
Merge pull request #425 from GorudenOsiris/patch-1

Create GorudenOsiris.md

ObedG committed:
Create Obed.md

dorian11-byte committed:
Create dorian11-byte.md

AngelZacarias committed:
MyQuote

My quote for the challenge

brauliogrc committed:
Create brauliogrc.md

Commit:
Merge pull request #430 from dorian11-byte/patch-1

Create dorian11-byte.md

devcifuentes committed:
Update devcifuentes.md

ObedG committed:
Rename Obed.md to ObedG.md

Commit:
Merge pull request #431 from AngelZacarias/patch-1

MyQuote

Commit:
Merge pull request #432 from ObedG/patch-1

Commit:
Merge pull request #428 from devcifuentes/patch-1

Creat devcifuentes.md

Commit:
Merge pull request #433 from brauliogrc/patch-2

Commit:
Merge pull request #410 from lissethamc/main

Alberto091003 committed:
Create Alberto091003.md

Llamas98mx committed:
Llamas98mx.md

Commit:
Merge pull request #438 from Alberto091003/patch-1

JsCasO committed:
Create JsCasO.md

Commit:
Merge pull request #444 from JsCasO/patch-1

Commit:
Merge pull request #445 from Llamas98mx/patch-2

torihendricks927 committed:
Create ToriHendricks.md

Creating for raffle

Commit:
Rename ToriHendricks.md to toriHendricks.md

Commit:
Merge pull request #461 from torihendricks927/patch-3

Create ToriHendricks.md

ramsessal committed:
Create ramsessal.md

FGVe1123 committed:
Create FGVe1123.md

Commit:
Merge pull request #464 from FGVe1123/patch-2

Create FGVe1123.md

Commit:
Merge pull request #462 from ramsessal/patch-3

Create ramsessal.md

skullDev21 committed:
Create skullDev21.md

Commit:
Merge pull request #466 from skullDev21/patch-2

Create skullDev21.md

fernandolpz21 committed:
Create fernandolpz21.md

Commit:
Merge pull request #471 from fernandolpz21/patch-2

Create fernandolpz21.md

Yohan-Petricoft committed:
Create Yohan-Petrift.md

Yohan-Petricoft committed:
Rename Yohan-Petrift.md to Yohan-Petricoft.md

zurdley committed:
Create zurdley.md

Commit:
Merge pull request #474 from Yohan-Petricoft/patch-1

Create Yohan-Petrift.md

Commit:
Merge pull request #479 from zurdley/main

Aletru committed:
Create Aletru.md

pamelasann committed:
Create pamelasann.md

Commit:
Merge pull request #483 from Aletru/patch-1

Commit:
Update pamelasann.md

Commit:
Merge pull request #484 from pamelasann/patch-1

Create pamelasann.md

RT-Isaac committed:
Create RT-Isaac.md

laisha1308 committed:
Create laisha1308.md

Raismaav committed:
Create Raismaav2510.md

Commit:
Merge pull request #490 from RT-Isaac/patch-7

Commit:
Merge pull request #492 from laisha1308/patch-4

Commit:
Merge pull request #493 from Raismaav2510/patch-2

ArthuR2D2 committed:
Create ArthuR2D2.md

Commit:
Merge pull request #499 from ArthuR2D2/patch-2

xsrpartyx committed:
Create xsrpartyx.md

Commit:
Merge pull request #500 from xsrpartyx/patch-1

Yoryo100 committed:
Create Yoryo100.md

Commit:
Merge pull request #502 from Yoryo100/patch-1

Create Yoryo100.md

louismanson committed:
Create louismanson.md

Add louismanson recommendation

4n0ph3 committed:
Create 4n0ph3

Commit:
Update 4n0ph3

Commit:
Merge pull request #508 from 4n0ph3/patch-1

Create 4n0ph3

GilbertoMV committed:
Create GilbertoMV.md

Commit:
Merge pull request #509 from GilbertoMV/patch-2

Create GilbertoMV.md

GimmoMx committed:
PullRequest

EduardoHGomez committed:
Create EduardoHGomez.md

Edgarhr2004 committed:
Create Edgarhr2004.md

magl76 committed:
Create magl76.md

PanyHyperx99 committed:
Create PanyHyperx99.md

Commit:
Merge pull request #514 from magl76/patch-1

Create magl76.md

Commit:
Merge pull request #515 from PanyHyperx99/patch-1

Create PanyHyperx99.md

Commit:
Merge pull request #513 from Edgarhr2004/patch-1

Create Edgarhr2004.md

Commit:
Merge pull request #512 from EduardoHGomez/patch-1

Create EduardoHGomez.md

Commit:
Merge pull request #511 from GimmoMx/main

PullRequest

Commit:
Merge pull request #506 from louismanson/patch-1

Add louismanson.md for talent-land

evangx committed:
Create evangx.md

Dinámica github

Commit:
Merge pull request #528 from evangx/patch-2

Create evangx.md

mikesholiu committed:
Create mikesholiu.md

LeslieSaucedo committed:
Merge pull request #537 from mikesholiu/patch-2

Create mikesholiu.md
